pkg/kind/kind.go

```diff
@@ -350,6 +454,12 @@ func parseKindVersion(v string) (float64, error) {
 	return floatVersion, nil
 }
 
-func deleteContainerRegistry() *exec.Cmd {
-	return exec.Command("docker", "rm", "-f", container_reg_name, "&&", "||", "true")
+func deleteContainerRegistry(dcli *dclient.Client) error {
+	if err := dcli.ContainerRemove(context.Background(), container_reg_name, container.RemoveOptions{Force: true}); err != nil {
```

Comment:
I think we need to check if a registry container is running first

Reply:
Why would it matter if the container was running or stopped?

Reply:
Sorry, shouldn't have used the word "running" in my original comment... the issue isn't whether the registry container is running or stopped but rather if it exists or not. 

Because the `deleteContainerRegistry()` method now returns an error, the whole quickstart will fail if the registry container doesn't exist.

```
$ ./kn-quickstart kind --registry
Running Knative Quickstart using Kind
✅ Checking dependencies...
    Kind version is: 0.23.0
💽 Installing local registry...
Pulling from library/registry: 2
Digest: sha256:ac0192b549007e22998eb74e8d8488dcfe70f1489520c3b144a6047ac5efbe90: %!s(<nil>)
Status: Image is up to date for registry:2: %!s(<nil>)
Error: creating cluster: failed to delete local registry: failed remove registry container: Error response from daemon: No such container: kind-registry
```

whereas before quickstart wouldn't fail when trying to delete a non-existent registry.

Reply:
Now I follow you. I will improve the error handling such that it does not return an error in this case instead of trying to check for the existence of the registry prior to deletion. It's possible that the registry container is deleted by another client prior to my delete container api call.